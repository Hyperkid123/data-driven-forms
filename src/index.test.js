import React from 'react';
import { mount } from 'enzyme'
import toJson from 'enzyme-to-json';
import Component from './index';

describe('Testing tests', () => {
  it('should pass', () => {
    expect(1).toEqual(1)
  });
  it('should render snapshot', () => {
    const wrapper = mount(<Component />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
});