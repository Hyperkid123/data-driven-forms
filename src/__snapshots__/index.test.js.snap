// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing tests should render snapshot 1`] = `
<DemoComponent>
  <div
    className="red-bg"
  >
    Foo component
  </div>
</DemoComponent>
`;
